articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,64 @@
+
+### Entanglement
+
+Possibly the most interesting phenomenon of quantum physics is the ability of two or more quantum particles to become **entangled** with each other. When particles become entangled, they form a single system such that the quantum state of any one particle cannot be described independently of the quantum state of the other particles. This means that whatever operation or process you apply to one particle correlates to the other particles as well, almost instantaneously.
+
+In addition to this interdependency, particles can maintain this connection even when separated over incredibly large distances, even light-years. The effects of quantum measurement also apply to entangled particles, such that when one particle is measured and collapses, the other particle collapses as well. Because there is a correlation between the entangled qubits, measuring the state of one qubit provides information about the state of the other qubit – this particular property is very helpful in quantum computing. These unique properties of entangled particles can be used to transmit quantum information from one place to another by a process called **teleportation**, and can play a role in some cryptography processes.
+
+### Qubits and probability
+
+Classical computers store and process information in bits, which can have a state of either 1 or 0, but never both. The equivalent in quantum computing is the **qubit**, which represents the state of a quantum particle. Because of superposition, qubits can either be 1 or 0, or both, or anything in between. Depending on its configuration, a qubit has a certain *probability* of collapsing to 1 or 0, and this probabilistic state is where the power of quantum computing excels.
+
+In classical computing, uncertainty is not a good thing. In quantum computing, uncertainty is built-in.
+
+For example, with two bits in a classical computer, each bit can store 1 or 0, so together you can store four possible values – **00**, **01**, **10**, and **11** – but only one of those at a time. With two qubits in superposition, however, each qubit can store 1 or 0 or *both*, so you can represent the same four values simultaneously. With three qubits, you can store eight values, with four qubits, you can store 16 values, and so on. Furthermore, when multiple entangled qubits are in superposition, they can process multiple options simultaneously.
+
+## Summary
+
+These concepts just scratch the surface of quantum physics, but are fundamentally important concepts to know for quantum computing.
```

Comment:
I'd say most of this isn't physics at all, but rather the basics of quantum mechanics.